app/views/nodes/_node_nearby.html.haml

```
@@ -1,26 +1,28 @@
 %section.node-nearby-locations-from-partners
-  %h2=t('.places-nearby')
+  %h2=t('.places_nearby')
   %section.node-nearby-locations-from-partners-list
 
 - content_for :javascript do
   :javascript
-    window.addEventListener("load", function load(event) {
-      window.removeEventListener("load", load, false); // remove listener, no longer needed
+    window.addEventListener("DOMContentLoaded", function load(event) {
+      window.removeEventListener("DOMContentLoaded", load, false); // remove listener, no longer needed
       AccessibilityCloud = window['accessibility.cloud'].default;
-      AccessibilityCloud.token = '7f039b60e27a4d02b13c5ad79fbe9d7b';
-      AccessibilityCloud.locale = I18n.locale;
+      var accessibilityCloud = new AccessibilityCloud({
+        token: '27be4b5216aced82122d7cf8f69e4a07',
+        locale: I18n.locale,
+      });
       var containerElement = document.querySelector('section.node-nearby-locations-from-partners');
       var element = document.querySelector('section.node-nearby-locations-from-partners-list');
       var parameters = {
         latitude: #{@node.lat},
         longitude: #{@node.lon},
-        accuracy: 1000,
-        limit: 20,
-        excludeSourceIds: 'LiBTS67TjmBcXdEmX', // excludes Wheelmap's own PoIs from results
+        accuracy: 1000, // include places within 1 km radius
+        limit: 10,
+        excludeSourceIds: ['LiBTS67TjmBcXdEmX'], // excludes Wheelmap's own PoIs from results
       };
-      AccessibilityCloud.loadAndRenderPlaces(element, parameters).done((response) => {
-        if (response && response.features && response.features.length) {
-          containerElement.className = containerElement.className += ' has-results';
+      accessibilityCloud.loadAndRenderPlaces(element, parameters, function(error, response) {
+        if (!error && response && response.features && response.features.length) {
```

1) Possible errors are broken responses and HTTP errors. Both cases fail silently here, the widget is simply not shown then. Users don't need to see them as the feature is just an addition, not core functionality of the page, but it would be nice if developers could see them, so if you have a solution for client-side error logging or monitoring, I'd include it here! How would I do this? Or could you even do it? ;)

2) The code is to ensure the widget's title is not shown when the list is empty (a successful response can be empty). The parent attribute extra check is for the case that the data structure is not as expected (should not happen, but you never know…), in this case I want to avoid an exception. The error parameter already carries the information if the response was successful or not.